helm-chart/renku/templates/_helpers.tpl

```diff
@@ -137,7 +137,7 @@ KC_DB_PASSWORD: {{ default (randAlphaNum 64) .Values.global.keycloak.postgresPas
 {{- end -}}
 
 {{- define "renku.gitlabUrl" -}}
-{{ .Values.global.gitlab.url | default (printf "%s://%s/gitlab" (include "renku.http" .) .Values.global.renku.domain) }}
+{{ .Values.global.gitlab.url | default (printf "%s://gitlab.%s/" (include "renku.http" .) .Values.global.renku.domain) }}
```

Comment:
do we need this? It's weird to have `renku.gitlabUrl` in some places and in others `.Values.global.gitlab.registry.host` for example. It would be cleaner maybe to just require `global.gitlab.url` to be set and not try to guess it.

Reply:
:man_shrugging: I don't really know, I thought this is still relevant when using the embedded gitlab. @SwissDataScienceCenter/yat do you know?

Reply:
I believe that will be done fully in https://github.com/SwissDataScienceCenter/renku/pull/3297 unless that's not relevant anymore?

Reply:
ah, I thought it was already merged...
IIRC, however, the internal Gitlab is broken anyway...
In any case, we can easily _still_ remove this value by replacing https://github.com/SwissDataScienceCenter/renku/blob/master/helm-chart/renku/templates/gateway/deployment.yaml#L50 with `global.gitlab.url` which will work for any deployment regardless.

Reply:
I've removed it